Development - Developed create label resource function and align with coding outline. (#3)
deshankoswatte/mit-iap-environment-platform: backend/services/src/github_services/Module.md

# Module Overview

## Description

This is the service for access github APIs'

## Utility Functions

### checkLabel

> The `checkLabel` function use to check whether the given label is available or not.

Reviewer: ```suggestion
The `checkLabel` function is used to check whether the given label is available or not.
```

- **Parameters**

>   - labelName - The checking label name.

Reviewer: ```suggestion
  - labelName - Name of the label.
```

> - **return** - The `checkLabel` function will return **json** to indicate the status.

Reviewer: ```suggestion
- **return** - The `checkLabel` function will return a **json** which indicates the status.
```

### createLabel

> The `createLabel` Function will create a label in github in particular repositary.

Reviewer: ```suggestion
The `createLabel` function will create a label in a specified git repository.
```

- **Parameters**

>   - labelName - The creating label name.

Reviewer: ```suggestion
  - labelName - Name of the label.
```

>   - labelDescription - The description of the label

Reviewer: ```suggestion
  - labelDescription - Description of the label.
```
